src/index.js

```diff
@@ -0,0 +1,159 @@
+const acceil = document.querySelector('.title');
+const searchArea = document.querySelector('.Search');
+const homecard = document.querySelector('.home_cards');
+const displayhomeItem = (result) => {
+  homecard.innerHTML = '';
+  result.forEach((item) => {
+    const scoreLi = document.createElement('div');
+    scoreLi.className = 'card';
+    scoreLi.innerHTML = `<img src="${item.image.medium}">
+                         <p class="movi-title">${item.name}</p>
+                         <div class='buttons-holder'>
+                         <button type="submit" class="comment-btn">Comment</button>
+                         <div class='likes-holder'>
+                         <i class="fa-regular fa-heart"></i><span class='likes-num'>0</span>
+                         </div>
+                         </div>`;
+    homecard.appendChild(scoreLi);
+  });
+  const cards = document.querySelectorAll('.card');
+  numberItem.textContent = cards.length;
+};
+
+const displayItem = (results) => {
+  card.innerHTML = '';
+  results.forEach((item) => {
+    const scoreLi = document.createElement('div');
+    scoreLi.className = 'card';
+    scoreLi.innerHTML = `<img src="${item.image.medium}">
+                         <p class="movi-title">${item.name}</p>
+                         <div class='buttons-holder'>
+                         <button type="submit" class="comment-btn">Comment</button>
+                         <div class='likes-holder'>
+                          <i class="fa-regular fa-heart"></i><span class='likes-num'>0</span>
+                          </div>
+                         </div>`;
+    card.appendChild(scoreLi);
+  });
+  const cards = document.querySelectorAll('.card');
+  numberItem.textContent = cards.length;
+};
+
+const backgroundBlur = () => {
+  const popup = document.querySelector('.popup');
+  const cover = document.querySelector('.cover');
+  const body = document.querySelector('body');
+  cover.classList.add('d-block');
+  body.style.overflow = 'hidden';
+  const closeBtn = document.querySelector('.fa-xmark');
+  closeBtn.addEventListener('click', () => {
+    popup.classList.add('d-none');
+    cover.classList.remove('d-block');
+    body.style.overflowY = 'scroll';
+  });
+};
+
+const displayPopUp = async (id) => {
+  const popup = document.querySelector('.popup');
+  await getShow(id).then((data) => {
+    popup.innerHTML = '';
+    popup.classList.remove('d-none');
+    popup.style.width = '80%';
+    popup.style.zIndex = '200';
+    renderPopUp(temp(data));
+    backgroundBlur();
+    displayComment(id);
+  });
+};
+
+const searchShow = async (query) => {
+  const BASE_URL = `https://api.tvmaze.com/search/shows?q=${query}`;
+  await fetch(BASE_URL)
+    .then((respose) => respose.json())
+    .then((jsonData) => {
+      const results = jsonData.map((item) => item.show);
+      displayItem(results);
+      updateLikes(results);
+      const commentBtn = document.querySelectorAll('.comment-btn');
```

Comment:
- [ ] In evaluating `Promises`, it is best practice to use either `async and await` or `thenable chain` but not both at the same time. If you remove the `async and await`, the code will still work. In this project you are required to use `async and await` , so please ensure to stick to the requirements.

Reply:
@AnselemOdims, thank you for the review. I am sorry, I do not understand what you mean, doesn't the async and await return a promise. Please how do I handle the promise?

Reply:
```
const response = await fetch(BASE_URL)
const data = await response.json()
const result = data.map((item) =>....)
```

Reply:
Oh thank you very much